analyzers/src/SonarAnalyzer.VisualBasic/Rules/ObsoleteAttributes.cs

```diff
@@ -24,4 +24,10 @@ namespace SonarAnalyzer.Rules.VisualBasic;
 public sealed class ObsoleteAttributes : ObsoleteAttributesBase<SyntaxKind>
 {
     protected override ILanguageFacade<SyntaxKind> Language => VisualBasicFacade.Instance;
+
+    protected override SyntaxNode GetExplanationExpression(SyntaxNode node) =>
+       node is AttributeSyntax { ArgumentList.Arguments: { Count: >= 1 } arguments }
```

Comment:
You don't need to fix this as it should be done as part of the rework of the ParameterLookup, but for Obsolete it is not as easy:
```cs
    [Obsolete(error: true, message: "Test")]
```
We can live with the false negative here for the moment.